src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -71,9 +73,10 @@ public static GraphBuilder create(
     boolean hasOsm = dataSources.has(OSM);
     boolean hasGtfs = dataSources.has(GTFS);
     boolean hasNetex = dataSources.has(NETEX);
-    boolean hasTransitData = hasGtfs || hasNetex;
+    boolean hasTransitDataSets = hasGtfs || hasNetex;
```

Comment:
I am not sure if `Set` add anything here, to me it become confusing since the other flags are also sets. I also think it is irrelevant if there are one or more data sources.

Reply:
I'm fine with reverting.

Reply:
I think the data set more clearly refers data packages being included while data could either refer to that or that there is actual data within the data sets (which this doesn't really check)

Reply:
How about I rename these as `hasOsmSources`, `hasGtfsSources`, `hasNetexSources` and `hasTransitDataSources`? Although, I'm not completely opposed to reverting the naming back to just `hasTransitData` either

Reply:
Either or works, but keep in mind that these are local variables and can have short names. I vote for keeping the short names, but do not really have a strong opinion - except being consistent.

Reply:
I ended up reverting the renaming as it is quite clear in this scope what the variable means